### PR TITLE
Integrate Travis using industrial_ci.

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

### CODE REVIEW

**Comment by mathias-luedtke** [comment]
Fixed `industrial_experimental` packages in https://github.com/ros-industrial/industrial_experimental/pull/13

**Comment by mathias-luedtke** [comment]
`BEFORE_SCRIPT='touch $CATKIN_WORKSPACE/src/industrial_experimental/IRC_v2/CATKIN_IGNORE'` could work

**Comment by gavanderhoorn** [comment]
Ok, something weird in [build 24](https://travis-ci.org/ros-industrial/motoman_experimental/builds/229676556):

> ROS distro '' is not supported

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+matrix:
+  allow_failures:
+    - env: ROS_DISTRO="indigo"  ROS_REPO=ros               UPSTREAM_WORKSPACE=file
+    - env: ROS_DISTRO="indigo"  ROS_REPO=ros-shadow-fixed  UPSTREAM_WORKSPACE=file
+
+install:
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+
+env:
```
Comment (mathias-luedtke) [inline .travis.yml]: this seems to overwrite the other block

Comment (gavanderhoorn) [reply .travis.yml]: hm, I assumed it was a 'merge dictionary' thing. Let's see if this works better.

**Comment by gavanderhoorn** [comment]
Ok. Fixed the build errors in the pkgs in this repository. Travis build should now succeed. Let's wait for it to complete.

**Comment by gavanderhoorn** [comment]
Ok. All checks passed ✔️.

**Comment by gavanderhoorn** [comment]
Thanks @ipa-mdl for your assistance. Invaluable as always.

**Pull request merged**